app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide :title, t(".title") %>
+<h1 class="center" style="color: blue; font-size: 50px;"> Help </h1>
+<div style="margin-left: 50px; font-size: 20px; " class="center">
+   <p> <%= t(".descripiton", link: link_to(t("about_rails"), "http://www.railstutorial.org/help/")).html_safe %></p>
```

Comment:
file này check lại index 

cho chị hỏi `html_safe` nớ là j thế

Reply:
cái này theo em hiểu thì thêm vào để bảo đảm an toàn, không cho chèn hay viết thêm code vào đoạn mã đó

Reply:
Sai indent
1 tab =  2 space thôi

self check nhé